hamocc/mo_carchm.F90

```diff
@@ -310,6 +311,7 @@ subroutine carchm(kpie,kpje,kpke,kbnd,pdlxp,pdlyp,pddpo,prho,pglat,omask,psicomo
                 ! Schmidt number water phase
                 sch_nh3_w  = mu_w   /(diff_nh3_w * rrho * 1000.)
               endif
+              
```

Comment:
Here and below (l 354): not sure, if you added or deleted whitespaces.

Reply:
I think I added a blank line for better readability